lang/en.js

```diff
@@ -68,9 +69,10 @@ export default {
   "showMore": "Show more",
   "record": "Record",
   "messages": {
+    "externalContentError": "The external content failed to load",
     "notFound": "Not Found",
     "noTitle": "No title provided for \"{record}\"",
```

Comment:
this "no title provided for..." string is no longer used.
If we don't want to show it anymore we can get rid of the string.
Otherwise it'll need to be added back to the search result.

Reply:
Removed from the lang files in 5263e4a.